auth-web/src/components/auth/create-account/AccountCreatePremium.vue

```diff
@@ -321,25 +320,17 @@ export default class AccountCreatePremium extends Mixins(Steppable) {
     this.isBaseAddressValid = !!isValid
   }
 
-  private get enableOrgNameAutoComplete (): boolean {
-    return LaunchDarklyService.getFlag(LDFlags.EnableOrgNameAutoComplete) || false
-  }
-
-  private setAutoCompleteSearchValue (autoCompleteSearchValue: string): void {
-    if (this.enableOrgNameAutoComplete) {
-      this.autoCompleteIsActive = false
-      this.orgName = autoCompleteSearchValue
-    }
+  private updateOrgBusinessType (orgBusinessType: OrgBusinessType) {
+    this.orgBusinessTypeLocal = JSON.parse(JSON.stringify(orgBusinessType))
```

Comment:
can we make all this as one action and set everything inside?

Reply:
Yeah :)